prime-router/docs/getting_started.md

```diff
@@ -386,22 +366,17 @@ cd ./prime-router
 
 When invoked with `--prune-volumes`, this script will also reset your PostgreSQL database. This can be useful to get back to a known and/or empty state.
 
-## Resetting just your database
-
-You can also use [Flyway](https://flywaydb.org/) to reset your database:
-```bash
-# drop and recreate the database; you may be prompted for a password
-export PGHOST=localhost
-export PGUSER=prime
-export PGDATABASE=prime_data_hub
-dropdb --force ${PGDATABASE?}
-createdb --owner=${PGUSER?} ${PGDATABASE?}
-
-# migrate the local database by hand
-flyway "-user=${PGUSER?}" -password=changeIT! \
-    -url=jdbc:postgresql://${PGHOST?}:5432/${PGDATABASE?} \
-    -locations=filesystem:./src/main/resources/db/migration migrate
-```
+## Resetting the Database
+1. Stop your ReportStream container if it is running.
+    ```bash
+    docker compose down
```

Comment:
I'd stick to using `docker-compose` as opposed to `docker compose`.

Reply:
Hmmm... This is the message from docker-compose:
Docker Compose is now in the Docker CLI, try `docker compose`

Reply:
`docker compose` is v2 of compose (rewritten in Go), whereas `docker-compose` defaults to the python-based compose v1. Further testing may be needed to ensure that v2 works with our setup (see `docker-compose version` vs `docker compose version`).

Reply:
I looked a bit more into this.  It looks like they just created a replacement to the tool regardless of the version of the files.  So, it really is the same command with just the Dash removed.  The one thing I could not find is where is this command supported and some text leads me to believe it may only work for Docker Desktop in Windows or Mac.  I'll switch this text to use the dash, but I am curious if Linux has this change too.
`The Docker CLI now supports the compose command, including most of the docker-compose features and flags, without the need for a separate tool. You can replace the dash (-) with a space when you use docker-compose to switch over to docker compose. You can also use them interchangeably, so that you are not locked-in with the new compose command and, if needed, you can still use docker-compose.`  - https://collabnix.com/docker-compose-now-shipped-with-docker-by-default/